src/main/java/io/zksync/transaction/type/Transaction712.java

```diff
@@ -79,8 +79,8 @@ public List<RlpType> asRlpValues(Sign.SignatureData signatureData) {
 
         result.add(RlpString.create(Numeric.hexStringToByteArray(getFrom()))); // 11
 
-        BigInteger ergsPerPubdata = getErgsPerPubdata();
```

Comment:
Just wanted to outline (wou'll probably fix it in subequent PRs, but still) that such changes cause test failures during transaction encoding (e.g. in https://github.com/zksync-sdk/zksync2-java/blob/breaking-changes/src/test/java/io/zksync/transaction/Transaction712Test.java).